assets/readme.md

```diff
@@ -1,19 +1,121 @@
+# Hull Clearbit Connector
 
-Fetch 3rd party data from Clearbit and writes it back into the customer profile.
+The Clearbit Connector enables sales and marketing teams to enrich customer profiles, reveal anonymous website traffic, find the ideal buyer and discover new accounts.
 
-# Setup
+## Getting Started
 
-- Go to your [Clearbit dashboard, under API keys](https://dashboard.clearbit.com/keys). Copy the key, Paste it into your Hull Dashboard.
-- Go to your [Clearbit Account Settings](https://dashboard.clearbit.com/account). Under Webhook URL, enter https://hull-clearbit.herokuapp.com/clearbit
+Go to the Connectors page of your Hull organization, click the button “Add Connector” and click “Install” on the Clearbit card. After installation, switch to the “Settings” tab and begin with the configuration.
 
+Begin your configuration in the section **Authentication** by copying the Clearbit API key into the provided field. If you are not sure which key to use, see [Find Your Clearbit API Key](#Find-Your-Clearbit-API-Key) for details.
+![Getting Started Step 1](./docs/gettingstarted01.png)
+Your Clearbit Connector is now ready to be configured depending on your needs, see Features to learn how to configure the various solutions.
 
-## Similar Lead discovery
+## Features
 
-The lead discovery feature uses [Clearbit Discovery](https://clearbit.com/discovery) and [Clearbit Prospector](https://clearbit.com/prospector) to find new leads based your existing users.
+Clearbit [**Enrichment**](#Enrichment) enables your team to transform the email address of an user in Hull into a complete user and company profile with up to 85 data points.
 
-Whenever a new user enters a segment that has been enabled in the "Similar lead discovery" section, this ship will :
+With Clearbit [**Reveal**](#Reveal) you can turn your anonymous website traffic into company profiles to power targeted campaigns and close more named accounts .
 
-1. Look for companies that are similar to the user's company via [Clearbit's Discovery](https://clearbit.com/discovery)
-2. Then for each Company found, we look for Prospect that match a specified role and/or seniority within that Company, via [Clearbit's Prospector](https://clearbit.com/prospector)
-3. For each Prospect found, we create a new User on your organization with the "Cleabit > Prospected at" attribute set to the current date.
+Use the [**Prospector**](#Prospector) to grow your pipeline and find the right buyer within a named account; all you need is a domain name. Our connector enables your team to automatically send revealed companies back to Clearbit and obtain targeted user profiles, no manual work required.
+
+[**Discover**](#Discover) new accounts that are similar to the profiles in any segment defined in Hull. Leverage our advanced segmentation engine to search through over 20M companies.
+
+The Clearbit connector supports to `create users`, `add traits` and `update traits`.
```

Comment:
@sbellity, again this is the feature description from Clearbit, what is meant with "close more named accounts" is that by using clearbit reveal, your sales team turn anonymous (also referred to as nameless) accounts into named/identified accounts which helps to close them. This is sales terminology, so I have decided not to mess with it and keep it for granted.